src/codex_plus/llm_execution_middleware.py

```
@@ -34,25 +34,54 @@
+
+        self.cerebras_middleware = CerebrasMiddleware()
         self.claude_dir = self._find_claude_dir()
         self.commands_dir = self.claude_dir / "commands" if self.claude_dir else None
         self.codexplus_dir = Path(".codexplus/commands")
         self.home_codexplus_dir = Path.home() / ".codexplus" / "commands"
+
+    def _is_cerebras_upstream(self, url: str) -> bool:
+        """Check if URL is Cerebras API endpoint"""
+        try:
+            parsed = urlparse(url)
+        except Exception:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)